app/controllers/obituaries/email_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Obituaries::EmailController < ApplicationController
+    def new 
+        @obit = Obituary.find(params[:id])
+    end 
+
+    def create 
+        recipient = params[:recipient_email]
+        recipient_name = params[:recipient_name]
+        sender = current_user
+        obit_id = params[:id]
+        ShareObituaryMailer.inform(recipient, recipient_name, sender, obit_id).deliver_now 
+        redirect_to "/obituaries/#{obit_id}"
+        flash[:notice] = "You Have Successfully Shared this Obituary in Email"
```

Comment:
Would like to learn how to make this flash message pretty.

Reply:
I think it will automatically be pretty because of what we dropped into layouts folder! We'll see :) If not, we can make sure it has all the right stuff to be included in that global styling.